pom.xml

```diff
@@ -117,13 +117,19 @@
         <modular.jdk.args/>
         <surefire.system.args>${modular.jdk.args}</surefire.system.args>
         <automatic.module.name>${project.groupId}.${project.artifactId}</automatic.module.name>
+
+        <!-- Used in SE bom -->
+        <version.org.codehaus.woodstox.woodstox-core-asl>4.4.1</version.org.codehaus.woodstox.woodstox-core-asl>
+        <version.org.apache.qpid>0.10.0</version.org.apache.qpid>
```

Comment:
same as above, these 2 props are not used by most users and I think should not be in BOM.

Reply:
@chengfang changes modified. Thanks.